test/conftest.py

```diff
@@ -22,11 +23,9 @@
 from utils import start_instances
 from utils import DEFAULT_RUN_DIR
 
-
-FILES_DIR = 'test/files'
-INIT_NO_CARTRIDGE_FILEPATH = os.path.join(FILES_DIR, 'init_no_cartridge.lua')
-INIT_IGNORE_SIGTERM_FILEPATH = os.path.join(FILES_DIR, 'init_ignore_sigterm.lua')
-INIT_ADMIN_FUNCS_FILEPATH = os.path.join(FILES_DIR, 'init_admin_funcs.lua')
+from project import INIT_NO_CARTRIDGE_FILEPATH
+from project import INIT_IGNORE_SIGTERM_FILEPATH
+from project import INIT_ADMIN_FUNCS_FILEPATH
```

Comment:
I'm not sure, but PEP8 doesn't welcome such imports. 

```python
from project import (INIT_NO_CARTRIDGE_FILEPATH, 
     INIT_IGNORE_SIGTERM_FILEPATH, ...)
```

Maybe something like this?

Reply:
I like your approach, but I suggest you to prettify our tests in separated patch. Please, create an issue and describe these problems.